packages/xforms-engine/src/lib/reactivity/createItemCollection.ts

```diff
@@ -101,6 +102,8 @@ const createItemsetItemLabel = (
 interface ItemsetItem {
```

Comment:
If rank and `<item>` require geometry and metadata, then we can use only this interface and remove BaseItem.

I think `<item>` needs geometry and metadata, but rank does not.